src/cli/cli/attach_cmd.rs

```diff
@@ -225,18 +225,15 @@ async fn send_message(
 ) -> Result<MpoolPushMessageResult, jsonrpc_v2::Error> {
     let (from, to, value) = params;
 
-    let value = humantoken::parse(&value)?;
+    let message = Message::transfer(
+        Address::from_str(&from)?,
+        Address::from_str(&to)?,
+        humantoken::parse(&value)?, // Convert forest_shim::TokenAmount to TokenAmount3
+        METHOD_SEND,
+        0,
```

Comment:
The `method_num` and `gas_limit` can be set by `Message::transfer`.

Reply:
Using the defaults?

Reply:
Well, setting the method number to `METHOD_SEND` is inherently part of transferring FIL. It's not just a default.